account-management-api/src/main/java/uk/gov/di/accountmanagement/domain/AccountManagementAuditableEvent.java

```diff
@@ -8,7 +8,9 @@ public enum AccountManagementAuditableEvent implements AuditableEvent {
     UPDATE_PHONE_NUMBER,
     ACCOUNT_MANAGEMENT_AUTHENTICATE,
     DELETE_ACCOUNT,
-    SEND_OTP;
+    SEND_OTP,
+    ACCOUNT_TEMPORARILY_LOCKED,
```

Comment:
Were these audit events agreed with someone? Was just looking at frontend-api audit events and can't see an analogous one: https://github.com/alphagov/di-authentication-api/blob/d6d39776743a5be555ea45dd59f0a6e389cf2ea4/frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/domain/FrontendAuditableEvent.java

Not to say this is wrong, but maybe worth double checking what events to send if not done already